config/manager/kustomization.yaml

```diff
@@ -16,5 +16,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: splunk/splunk-operator
-  newTag: 1.1.0
\ No newline at end of file
+  newName: 667741767953.dkr.ecr.us-west-2.amazonaws.com/mgaldino/splunk/splunk-operator
```

Comment:
I don't think so this is correct. CC: @vivekr-splunk ?

Reply:
This was reminiscent of the testing. removing in the next commit.